jena-arq/src/main/java/org/apache/jena/riot/RDFParserBuilder.java

```diff
@@ -0,0 +1,431 @@
+     * or HTTP Content-Type.
+     * @see Lang
+     * @param lang
+     * @return this
+     */
+    public RDFParserBuilder forceLang(Lang lang) { this.forceLang = lang ; return this; }
+    
+    /**
+     * Set the HTTP "Accept" header.
+     * The default if not set is {@link WebContent#defaultRDFAcceptHeader}.
+     * @param acceptHeader
+     * @return this
+     */
+    public RDFParserBuilder httpAccept(String acceptHeader) { 
+        httpHeader(HttpNames.hAccept, acceptHeader);
+        return this; 
+    }
+
+    /**
+     * Set an HTTP header. Any previous setting is
+     * <p> 
+     * Consider setting up an {@link HttpClient} if more complicated setting to an HTTP
+     * request is required.
+     */
+    public RDFParserBuilder httpHeader(String header, String value) {
+        httpHeaders.put(header, value);
+        return this;
+    }
+    
+    /** Set the HttpClient to use.
+     *  This will override any HTTP header settings.
+     */
+    public RDFParserBuilder httpClient(HttpClient httpClient) {
+        this.httpClient = httpClient;
+        return this;
+    }
+
+    public RDFParserBuilder base(String base) { this.baseUri = base ; return this; }
+
+    /**
+     * Set the {@link ErrorHandler} to use.
+     * This replaces any previous setting.
+     * The default is use slf4j logger "RIOT".   
+     * @param handler
+     * @return this
+     */
+    public RDFParserBuilder errorHandler(ErrorHandler handler) {
+        this.errorHandler = handler;
+        return this;
+    }
+    
+    /**
+     * Set the {@link FactoryRDF} to use. {@link FactoryRDF} control how parser output is
+     * turned into {@code Node} and how {@code Triple}s and {@code Quad}s are built. This
+     * replaces any previous setting. 
+     * <br/>
+     * The default is use {@link RiotLib#factoryRDF()} which is provides {@code Node}
+     * reuse. 
+     * <br/>
+     * The {@code FactoryRDF} also determines how blank node labels in RDF syntax are
+     * mapped to {@link BlankNodeId}. Use
+     * <pre>
+     *    new Factory(myLabelToNode) 
+     * </pre>
+     * to create an {@code FactoryRDF} and set the {@code LabelToNode} step.
+     * @see #labelToNode
+     * @param factory
+     * @return this
+     */
+    public RDFParserBuilder factory(FactoryRDF factory) {
+        this.factory = factory;
+        return this;
+    }
+    
+    /**
+     * Use the given {@link LabelToNode}, the policy for converting blank node labels in
+     * RDF syntax to Jena's {@code Node} objects (usually a blank node).
+     * <br/>
+     * Only applies when the {@link FactoryRDF} is not set in the
+     * {@code RDFParserBuilder}, otherwise the {@link FactoryRDF} controls the
+     * label-to-node process.
+     * <br/>
+     * {@link SyntaxLabels#createLabelToNode} is the default policy.
+     * <br>
+     * {@link LabelToNode#createUseLabelAsGiven()} uses the label in teh RDF syntax directly. 
```

Comment:
Thanks - I've fixed these in the latest push which includes deprecations in `RDFDataMgr`.